Gemfile

```
@@ -2,3 +2,4 @@ source 'https://rubygems.org'
 gem "nokogiri", "~> 1.10"
 gem "jekyll", "~> 4.0"
 gem "jekyll-last-modified-at", "~> 1.1.0"
+gem 'jekyll-redirect-from'
```

Let's pin this like we do the other Ruby gems. This is [the latest version](https://rubygems.org/gems/jekyll-redirect-from) and also the version used [when you deploy](https://app.netlify.com/sites/cfpb-design-system/deploys/618018cecb45900007b794f8).

```suggestion
gem "jekyll-redirect-from", "~> 0.16.0"
```